src/main/java/de/wejago/hichi2influx/service/Subscriber.java

```diff
@@ -0,0 +1,6 @@
+package de.wejago.hichi2influx.service;
+
+import org.eclipse.paho.client.mqttv3.IMqttMessageListener;
```

Comment:
You can remove that interface and let JsonSubscriber directly implement IMqttMessageListener

Reply:
My understanding is that the next step is to add `RawSubscriber` that will extend `Subscriber` and will have to implement this interface. That's why I added it in the parent class.